DotNetHowToIndexers/DotNetHowToIndexers.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Again, to be consistent with other examples on this page, we should probably target .net framework 4.5 instead of .net core for now

Reply:
At least one of the other samples is based on .NET Core, and that is the direction we should be moving in. I'm fine with this sample targeting .NET Core as long as the instructions for building/running the sample are clear.

Reply:
I agree with Bruce, I would love to use this as a basis of a demo to run in Docker on Windows which would need .NET core

Reply:
I left it as .NET Core